spec/Section 6 -- Execution.md

```diff
@@ -17,14 +17,16 @@ operations” section.
 ## Evaluating operations
 
 The type system, as described in the “Type System” part of the spec, must
-provide a “Query Root” and a “Mutation Root” object.
+provide a “Query Root” and a “Mutation Root” object, and can provide an optional
```

Comment:
This existing phrase isn't quite correct. Do you mind amending to:

The type system, as described in the "Type System" part of the spec, must provide a "Query Root" and may optionally also provide a “Mutation Root” or a "Subscription Root" object.